mso/resource_mso_schema_site_external_epg.go

```diff
@@ -56,6 +56,23 @@ func resourceMSOSchemaSiteExternalEpg() *schema.Resource {
 				Computed:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1000),
 			},
+			"l3out_template": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"l3out_schema": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"l3out_on_apic": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
```

Comment:
1. handle the read and import of these attributes. 
2. adjust the datasource for these attributes.
3. update the example to include a APIC and a NDO l3out
4. update documentation with attributes missing ( website/docs/r/mso_schema_site_external_epg.html.markdown )